.github/CODEOWNERS

```diff
@@ -1934,3 +1934,4 @@ spec/fixtures/va_profile/health_benefit_v1_associated_persons.json @department-o
 spec/lib/va_profile/health_benefit/service_spec.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-health-modernization-initiative @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/logging/third_party_transaction.rb @department-of-veterans-affairs/backend-review-group
 spec/lib/logging/third_party_transaction_spec.rb @department-of-veterans-affairs/backend-review-group
+app/models/lighthouse_supporting_evidence_attachment.rb @department-of-veterans-affairs/backend-review-group
```

Comment:
@NB28VT – I'm not certain on the protocol for updates to this file, but it appears, for the most part, that additions are listed alphabetically by file path.